code/game/response_team.dm

```diff
@@ -112,7 +112,7 @@ var/send_emergency_team
 				return
 			var/client/C = src.client
 			var/mob/living/carbon/human/new_commando = C.create_response_team(L.loc)
-			del(L) // Do not qdel landmarks!
```

Comment:
yes, landmarks don't qdel well, at all, but there's really no point in directly stating they don't qdel well, when it's implied with the landmark's Destroy(), which now returns a hint of HARDDEL_NOW.